app/src/main/java/com/codingblocks/cbonlineapp/activities/HomeActivity.kt

```diff
@@ -400,7 +372,7 @@ class HomeActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelecte
         shortcutManager.removeAllDynamicShortcuts()
     }
 
-    fun isNetworkAvailable(context: Context): Boolean {
+    private fun isNetworkAvailable(context: Context): Boolean {
```

Comment:
this function can be moved to ViewModel as well

Reply:
or you may move this function to utils class as well as delete database function

Reply:
We cannot move this to viewmodel. As viewmodel never has android imports, I shall move it to utils instead. A proper connection class

Reply:
@aggarwalpulkit596 check